### PR TITLE
Add BRNKC Mainnet and BRNKC Testnet

### PR DESCRIPTION
**Pull request by BearNetwork-BRNKC**
## Description

_Concise description of proposed changes_

## Additional Information

- [x] I read the [contributing docs](/wagmi-dev/references/blob/main/.github/CONTRIBUTING.md) (if this is your first contribution)

Your ENS/address:

### CODE REVIEW

File: .changeset/eleven-trainers-yell.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@wagmi/chains': minor
```
Comment (tmm) [inline .changeset/eleven-trainers-yell.md:2]:
```suggestion
'@wagmi/chains': patch
```

File: .changeset/eleven-trainers-yell.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+---
+
+Add BearNetworkChainMainnet and BearNetworkChainTestnet
```
Comment (tmm) [inline .changeset/eleven-trainers-yell.md:5]:
```suggestion
Added Bear Network Mainnet  and Testnet
```

File: packages/chains/src/BearNetworkChainTestnet.ts [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    symbol: 'tBRNKC',
+  },
+  rpcUrls: {
+    public: { http: ['https://brnkc-test.bearnetwork.net'] },
+    default: { http: ['https://brnkc-test.bearnetwork.net'] },
+  },
+  blockExplorers: {
+    default: {
+      name: 'BrnkTestScan',
+      url: 'https://brnktest-scan.bearnetwork.net/',
```
Comment (tmm) [inline packages/chains/src/BearNetworkChainTestnet.ts:19]:
```suggestion
      url: 'https://brnktest-scan.bearnetwork.net',
```

**Comment by BearNetwork-BRNKC** [comment]
@tmm Thank you very much. We have confirmed the changes.

File: .changeset/eleven-trainers-yell.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+---
+'@wagmi/chains': patch
+---
+
+Added Bear Network Mainnet  and Testnet
```
Comment (tmm) [inline .changeset/eleven-trainers-yell.md:5]:
```suggestion
Added Bear Network Mainnet and Testnet
```

**Review by tmm — COMMENTED** [review]
You'll also want to export the chains from the `index.ts` file.

File: packages/chains/src/index.ts [hunk 5]
```diff
@@ -98,3 +98,5 @@ export { xdcTestnet } from './xdcTestnet'
 export { zhejiang } from './zhejiang'
 export { zkSync } from './zkSync'
 export { zkSyncTestnet } from './zkSyncTestnet'
+export { brnkc } from './BearNetworkChainMainnet'
+export { brnkcTestnet } from './BearNetworkChainTestnet'
```
Comment (tmm) [inline packages/chains/src/index.ts]: Please alphabetize

File: packages/chains/src/index.ts [hunk 6]
```diff
@@ -98,3 +98,5 @@ export { xdcTestnet } from './xdcTestnet'
 export { zhejiang } from './zhejiang'
 export { zkSync } from './zkSync'
 export { zkSyncTestnet } from './zkSyncTestnet'
+export { BearNetworkChainMainnet } from './BearNetworkChainMainnet'
+export { BearNetworkChainTestnet } from './BearNetworkChainTestnet'
```
Comment (tmm) [inline packages/chains/src/index.ts:102]: Please alphabetize. Also, missed this earlier: the chain names and file names should be lower camelcase (e.g. `bearNetworkChainMainnet`)

**Comment by BearNetwork-BRNKC** [comment]
@tmm Wow, Thank you very much for your guidance.

**Comment by BearNetwork-BRNKC** [comment]
@tmm Wow, I'm thrilled that it's done. I sincerely appreciate your guidance.

**Comment by BearNetwork-BRNKC** [comment]
@tmm That's fantastic! I greatly appreciate your assistance.

File: packages/chains/src/index.ts [hunk 7]
```diff
@@ -13,6 +13,8 @@ export { bronos } from './bronos'
 export { bronosTestnet } from './bronosTestnet'
 export { bsc } from './bsc'
 export { bscTestnet } from './bscTestnet'
+export { bearNetworkChainMainnet } from './bearNetworkChainMainnet'
+export { bearNetworkChainTestnet } from './bearNetworkChainTestnet'
```
Comment (tmm) [inline packages/chains/src/index.ts:17]: Should be below `baseGoerli` to be alphabetized

**Comment by BearNetwork-BRNKC** [comment]
@tmm I deeply appreciate your immense kindness and generosity. Thank you very much.

**Comment by BearNetwork-BRNKC** [comment]
@tmm I sincerely apologize for taking up so much of your valuable time. I am truly sorry for the inconvenience caused.

At the same time, I want to express my heartfelt gratitude for your willingness to spend time correcting our mistakes and successfully completing a series of adjustments.

You are truly a kind-hearted person.

File: .changeset/eleven-trainers-yell.md (hunk 1 above)
Comment (BearNetwork-BRNKC) [reply .changeset/eleven-trainers-yell.md]: been completed.

**Comment by BearNetwork-BRNKC** [comment]
@tmm 
May I ask if there are any other instructions or actions we need to take, such as clicking on something or making any modifications?

**Review by tmm — APPROVED** [review]

**Pull request merged**